### ISSUE TITLE
Healthchecks should be available

### ISSUE THREAD

**Issue by jkinkead** [Nordstrom/xrpc#26]
Feature request: Have healthchecks exposed somewhere.

See [Dropwizard's metrics API](http://metrics.dropwizard.io/3.2.3/manual/healthchecks.html).

**Comment by xjdr** [comment]
Currently, healthchecks are configurable like any other route:

https://github.com/Nordstrom/xrpc/blob/master/demo/src/main/java/com/nordstrom/xrpc/demo/Example.java#L161-L165

You could accomplish something similar to the link with something like:

```java
    Handler healthCheckHandler =
        context -> {
          final Database database = new Database();
          if (database.ping()) {
            return Recipes.newResponseOk("");
          } else {
            return Recipes.newResponseBadRequest("Can't ping database");
          }
        };
```

You would really want to have that database be a persistent connection that was used throughout the program, but the above should illustrate the point. Do you think that would be sufficient, or should we look to provide more structured health checks?

**Comment by xjdr** [comment]
Addressed by 0256e21a5168aab4493968d4792e897693da2fa7
